### PR TITLE
Added WindiCSS Badge

### PR DESCRIPTION
**Pull request by NawrasseDahman**

### CODE REVIEW

**Review by kohasummons — APPROVED** [review]
🍗 Looks good to me.

**Comment by NawrasseDahman** [comment]
Now Should it Get Merged Into The Main Branch Or Not

File: README.md [hunk 1]
```diff
@@ -367,6 +367,7 @@ Copy the appropriate `![Name](link)` element and paste it in your Markdown file
...
 | Vuetify            | ![Vuetify](https://img.shields.io/badge/Vuetify-1867C0?style=for-the-badge&logo=vuetify&logoColor=AEDDFF)                                       | `![Vuetify](https://img.shields.io/badge/Vuetify-1867C0?style=for-the-badge&logo=vuetify&logoColor=AEDDFF)`                                       |
 | Webpack            | ![Webpack](https://img.shields.io/badge/webpack-%238DD6F9.svg?style=for-the-badge&logo=webpack&logoColor=black)                                 | `![Webpack](https://img.shields.io/badge/webpack-%238DD6F9.svg?style=for-the-badge&logo=webpack&logoColor=black)`                                 |
+| WindiCSS            | ![Windicss](https://img.shields.io/badge/windicss-48B0F1.svg?style=for-the-badge&logo=windi-css&logoColor=white)                                 | `![Windicss](https://img.shields.io/badge/windicss-48B0F1.svg?style=for-the-badge&logo=windi-css&logoColor=white)`                                 |
```
Comment (Ileriayo) [inline README.md:370]: LGTM too!

**Review by Ileriayo — APPROVED** [review]
Thanks for contributing, @NawrasseDahman!

File: README.md (hunk 1 above)
Comment (NawrasseDahman) [reply README.md:370]: Np, just trying to improve this community better and if I  got any other badges i will be sure contribute them too

Comment (Ileriayo) [reply README.md:370]: Thanks again, @NawrasseDahman!

**Pull request merged**